app/models/helpers.rb

```diff
@@ -0,0 +1,11 @@
+class Helpers
```

Comment:
Consider adding these helper method in your ApplicationController rather than a model. 

```
helpers do 
  #helper methods here
end
```